node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log('STOP THAT');
```

Comment:
Here you want to log out whatever follows the file name when the node command is ran in the terminal. HINT: use `process.argv[2]`